java-api-core/src/com/kinvey/java/sync/SyncManager.java

```diff
@@ -159,35 +161,50 @@ public void enqueueRequest(SyncRequest request) {
         requestCache.save(request);
     }
 
-    public <T extends GenericJson> void enqueueRequest(String collectionName, NetworkManager<T> networkManager, RequestMethod method, String id) throws IOException {
+    public <T extends GenericJson> void enqueueRequest(String collectionName, NetworkManager<T> networkManager, SyncItem.HttpVerb httpMethod, String id) throws IOException {
         ICache<SyncItem> requestCache = cacheManager.getCache(SYNC_ITEM_TABLE_NAME, SyncItem.class, Long.MAX_VALUE);
```

Comment:
Are some of these API changes potentially breaking changes to the public API?

Reply:
We don't have any notes about this method or this class at the devcenter. It's class for internal SDK using. I'm not sure that we have to add this changes to `breaking changes` for release notes. These changes don't affect classes and methods which exist in devcenter notes.